modules/detection__disruption/main.py

```diff
@@ -23,7 +23,7 @@
     'services': ['GuardDuty', 'CloudTrail', 'EC2', 'Config', 'monitoring'],  # CloudWatch needs to be "monitoring" and VPC needs to be "EC2" here for "ls" to work
 
     # For prerequisite modules, try and see if any existing modules return the data that is required for your module before writing that code yourself, that way, session data can stay separated and modular.
-    'prerequisite_modules': ['detection__enumerate_services'],
```

Comment:
It won't let me comment on it, but `detection__enum_services` didn't have its module name updated in its code